src/js/components/FragmentList/fragmentList.jsx

```diff
@@ -16,43 +16,9 @@
 
 import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
-import styled from 'styled-components';
+import Styled from './fragmentList.style';
 import Fragment from './fragment';
 
-const StyledFragmentList = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: stretch;
-`;
-
-const SortingWrapper = styled.div`
-    display: flex;
-`;
-
-const SortingButton = styled.button`
-      border: 0;
-      margin: 2px;
-      padding: 2px;
-      color: ${({ theme }) => theme.textColor};
-      background-color: ${({ theme }) => theme.buttonBgColor};
-      &:active {
-        background-color: ${({ theme }) => theme.fragmentHighlight};
-      }
-`;
-
-const SortingButtonStatus = styled(SortingButton)`
-      width: 15px;
-`;
-
-const SortingButtonId = styled(SortingButton)`
-      flex: 1;
-`;
-
-const SortingButtonType = styled(SortingButton)`
-      flex: 1;
-`;
-
-
 const FragmentList = () => {
   function mapDataToComponents({ fragments }) {
```

Comment:
I would try to extract all of this functions from that component to helper methods and add their corresponding unit tests